### PR TITLE
Improve existing calendar documentation to be clearer

### PR DESCRIPTION
**Pull request by jedevc**
This patch updates the documentation for the calendar to be clearer and hopefully explains the process of what is going on.

I quite like the idea of using the table format to describe how to define each property, as it can then be used as a reference.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,46 @@ You can add a thumbnail to a post; simply link the thumbnail in the front
...
+implemented, so no other tags aside from the ones specified here should be
+used.
+
+| Content       | Source                                                                                                  |
+| :------------ | :------------------------------------------------------------------------------------------------------ |
+| title         | the name of it appears on the calendar                                                                  |
+| date          | the date and time as it appears on the calendar                                                         |
+| location      | the location as it appears on the calendar                                                              |
+| description   | `<desc></desc>` tags in the event description, otherwise, the full event description                    |
+| facebook link | `<fb></fb>` tags in the event description, otherwise, the first `<a></a>` tags in the event description |
```
Comment (LikkanChung) [inline README.md]: Not quite, for the link to be accessed from the "Facebook Event" button, it must be in the `<fb>` tag. The `<a>` tags are only removed if they match in the `<fb>` tags. 

i.e. for a link to be put into the button, it must be within the `<fb>` tag. The `<a>` tag is optional - caused by hyperlinks in the Google Calendar event

File: README.md [hunk 2]
```diff
@@ -63,6 +63,46 @@ You can add a thumbnail to a post; simply link the thumbnail in the front
...
+[FullCalendar][fullcalendar] and an API key managed from the Google Cloud
+Console.
+
+To add a new calendar event, a member of the CSS committee with access to the
+calendar must add a new event, and it will automatically appear on the
+website.
+
+#### Event formatting
+
+The event data displayed on the webiste is taken directly from the google
```
Comment (LikkanChung) [inline README.md]: Google Calendar - capital letters

File: README.md (hunk 1 above)
Comment (LikkanChung) [reply README.md]: Perhaps it wasn't quite clear enough in the code

Comment (jedevc) [reply README.md]: Ahhh alright, thanks for the clarification, will fix that.

**Comment by LikkanChung** [comment]
Great work. Thanks

**Pull request merged**